EnglishFilter/sections/foreign.txt

```diff
@@ -9295,6 +9295,8 @@ brandbuffet.in.th#@#.category-advertorial
 !------------------------------ Vietnam ------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/79151
+@@||static.doubleclick.net/instream/ad_status.js$domain=linkneverdie.net
```

Comment:
to antiadblock.txt please

Reply:
This is a Vietnamese Site.

Reply:
@wout-codes 

ABPVN filter causes antiadblock too so please add this rule:
`@@||tag.gammaplatform.com^$domain=linkneverdie.net`

and this for share button:
`linkneverdie.net##div[title="Chia sẻ"]`

Reply:
Okay.